tests/plugins/chart/basic_chart.test.ts

```diff
@@ -76,14 +76,14 @@ describe("datasource tests", function () {
       model,
       {
         dataSets: ["Sheet1!B1:B4", "Sheet1!C1:C4"],
-        labelRange: "Sheet1!A2:A4",
+        labelRange: "Sheet1!A1:A4",
```

Comment:
```suggestion
        labelRange: "Sheet1!A2:A4",
        dataSetsHaveTitle: true,
```
Considering the test data, it should be it, right ? Or I missed something ?

Reply:
> Considering the test data, it should be it, right ? Or I missed something ?

@pro-odoo Previously when the datasets have titles, the category XC will be automatically decreased by one, so if the dataset range is `B1:B4` including title, the label range will be changed into `A2:B4`. 
Now I cancel this automatic behavior and make `use row x as headers` to exclude the label header.